build.sbt

```diff
@@ -54,6 +54,7 @@ libraryDependencies ++= {
     "com.typesafe.akka"   %%  "akka-slf4j"    % akkaV,
     ("com.google.api-client" % "google-api-client" % "1.21.0").exclude("com.google.guava", "guava-jdk5"),
     "com.google.apis" % "google-api-services-storage" % "v1-rev58-1.21.0",
+    "com.google.apis" % "google-api-services-compute" % "v1-rev72-1.20.0",
```

Comment:
latest version is v1-rev120-1.22.0, according to https://cloud.google.com/compute/docs/api/libraries#google_apis_java_client_library